lib/view_model/restore/restore_from_qr_vm.dart

```diff
@@ -111,6 +111,7 @@ abstract class WalletRestorationFromQRVMBase extends WalletCreationVM with Store
               height: restoreWallet.height ?? 0,
               mnemonic: restoreWallet.mnemonicSeed ?? '',
               password: password,
+              passphrase: '',
```

Comment:
shouldn't this be also `passphrase ?? ''`

Reply:
correct